Releases.md

```diff
@@ -6,6 +6,44 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.32.0 / 2023.03.22
+
+- BREAKING(ext/fs): FileInfo.dev is defined on Windows (#18237)
+- BREAKING(unstable): remove WebGPU (#18094)
+- Revert "refactor: rename Deno.openKv() to Deno.kv() (#18349)" (#18362)
+- feat(cli): --ext parameter for run, compile, and bundle (#17172)
+- feat(compile): Add support for web workers in standalone mode (#17657)
+- feat(compile): Enable multiple roots for a standalone module graph (#17663)
+- feat(core): deno_core::extension! macro to simplify extension registration
+  (#18210)
+- feat(ext/kv): key-value store (#18232)
+- feat(ext/net): Add multicasting APIs to DatagramConn (#10706) (#17811)
+- feat(ext/url): URLSearchParams.size (#17884)
+- feat(fs): support FileInfo.dev on Windows (#18073)
```

Comment:
Let's remove this one as it's mentioned as the first `BREAKING(ext/fs)`

Reply:
I feel like we should remove the breaking one. It wasn't really breaking since it was a fix to a breaking.

Reply:
It's a breaking change because the typing changed from `null | number` to `number`

Reply:
It was `number | null` then `number` by accident (breaking) and now the fix is to restore the default behaviour, which I wouldn't really consider a breaking change, but just a fix.

Reply:
> It was number | null then number by accident (breaking) and now the fix is to restore the default behaviour, which I wouldn't really consider a breaking change, but just a fix.

No, it's still been changed for the `FileInfo.dev` fields, the remaining fields were reverted